packages/components/addon/components/hds/app-footer/link.hbs

```diff
@@ -0,0 +1,24 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{! TODO: use new Text component? }}
+<Hds::AppFooter::Item>
```

Comment:
From what I see this doesn't support leading/trailing icons. I _think_ this is an issue. 

(Not sure if it comes from design, or it's still a WIP, but I have a strong opinion we should support leading/trailing icons in this link: "guidance over control")

Reply:
If the intent is to let the consumers add them via the `{{yield}}`, I am not sure what would be the benefit? they have to do all the layout work, while we can do it for them "for free".

Reply:
Ok how do you propose we implement those if you think we should include them? Should I repurpose the Hds Standalone link or add custom arguments? (I didn't use the Standalone link since the links don't normally include icons and also I didn't see clear designs supporting an option for links when I added these originally.)

@andgen404 Do you have any suggestions or could you clarify the options we need to support?

Reply:
`Link::Inline` supports optional icons: https://helios.hashicorp.design/components/link/inline?tab=code#component-api. Depends on the design decision around 1) visual treatment for interactive states and 2) support for icons or not

Reply:
Ok thanks, I missed those. I exposed the arguments for now but I'll hold off adding any example to the Showcase for now since I didn't see a clear use case. (Let me know if there's a use case I'm missing or that you think would be useful.)